web-portal/components/SubmissionForm.vue

```diff
@@ -507,11 +507,12 @@
           this.showEventLoadingSpinner = false
           this.$emit('submitted')
         }).catch((error) => {
-          console.log(error)
+          console.error(`error uploading image: ${error}`)
+
```

Comment:
Might be better to log the error directly instead of interpolating it into a string -- avoid losing the stacktrace etc.
```suggestion
          console.error(`error uploading image:`, error)

```

Reply:
fixed e5e1f4f796f0615c9c76e2c4a63eb9154ee4f622